entity-framework/core/extensions/index.md

```diff
@@ -128,7 +128,7 @@ Easily perform temporal queries on your favourite database using introduced exte
 
 ### EFCore.TimeTraveler
 
-Allow full-featured Entity Framework Core queries against [SQL Server Temporal History](https://docs.microsoft.com/en-us/sql/relational-databases/tables/temporal-table-usage-scenarios?view=sql-server-ver15#point-in-time-analysis-time-travel) using the EF Core code, entities, and mappings you already have defined.  Travel through time by wrapping your code in `using (TemporalQuery.AsOf(targetDateTime)) {...}`. For EF Core: 3.
+Allow full-featured Entity Framework Core queries against [SQL Server Temporal History](https://docs.microsoft.com/sql/relational-databases/tables/temporal-table-usage-scenarios?view=sql-server-ver15#point-in-time-analysis-time-travel) using the EF Core code, entities, and mappings you already have defined.  Travel through time by wrapping your code in `using (TemporalQuery.AsOf(targetDateTime)) {...}`. For EF Core: 3.
```

Comment:
https://github.com/aspnet/EntityFramework.Docs/issues/1900#issue-518036263

Reply:
Makes sense @smitpatel, thanks. Should we put the linking guidelines somewhere in the [contribution guide](https://github.com/aspnet/EntityFramework.Docs/blob/master/CONTRIBUTING.md)?

Reply:
[They are already linked](https://github.com/aspnet/EntityFramework.Docs/blob/master/CONTRIBUTING.md#markdown-syntax)

Reply:
Thanks. Also removed the explicit linking to SQL Server 15 (now it just goes to the latest).